test/image/mocks/waterfall_profit-loss_2018vs2019_textinfo_base.json

```diff
@@ -0,0 +1,157 @@
+            ],
+            "text": [
+                "+ Sales",
+                "+ Consulting",
+                "+ Maintenance",
+                "+ Other revenue",
+                "= Net revenue",
+                "- Purchases",
+                "- Material expenses",
+                "- Personnel expenses",
+                "- Other expenses",
+                "= Operating profit",
+                "+ Investment income",
+                "+ Financial income",
+                "= Profit before tax",
+                "- Income tax (15%)",
+                "= Profit after tax"
+            ],
+            "x": [
+                375,
+                128,
+                78,
+                27,
+                null,
+                -327,
+                -12,
+                -78,
+                -12,
+                null,
+                32,
+                89,
+                null,
+                -45,
+                null
+            ],
+            "base": 0,
+            "textinfo": "text+delta",
+            "textposition": "outside"
+        },
+        {
+            "name": "2019",
+            "type": "waterfall",
+            "orientation": "h",
+            "measure": [
+                "relative",
+                "relative",
+                "relative",
+                "relative",
+                "total",
+                "relative",
+                "relative",
+                "relative",
+                "relative",
+                "total",
+                "relative",
+                "relative",
+                "total",
+                "relative",
+                "total"
+            ],
+            "y": [
+                "Sales",
+                "Consulting",
+                "Maintenance",
+                "Other revenue",
+                "Net revenue",
+                "Purchases",
+                "Material expenses",
+                "Personnel expenses",
+                "Other expenses",
+                "Operating profit",
+                "Investment income",
+                "Financial income",
+                "Profit before tax",
+                "Income tax (15%)",
+                "Profit after tax"
+            ],
+            "x": [
+                307,
+                102,
+                187,
+                172,
+                null,
+                -302,
+                -121,
+                -187,
+                -121,
+                null,
+                123,
+                198,
+                null,
+                -53.7,
+                null
+            ],
+            "base": 255,
+            "textinfo": "initial+final",
+            "textposition": "outside"
+        }
+    ],
+    "layout": {
+        "title": {
+            "text": "Profit and loss statement 2018 vs 2019<br>waterfall chart"
+        },
+        "yaxis": {
+            "type": "category",
+            "autorange": "reversed"
+        },
+        "xaxis": {
+            "type": "linear"
```

Comment:
Can you try adding `ticksuffix` and `tickprefix` to these axes to see if things work as planned?

Reply:
Good call.
Done in the `funnel` PR https://github.com/plotly/plotly.js/pull/3817/commits/6a3a13c6f20348a5637eaa87a81d07eb28125145.